addons/field_rations/$PBOPREFIX$

```diff
@@ -0,0 +1 @@
+z\acex\addons\field_rations
```

Comment:
Empty newline missing.

Reply:
I thought `$PBOPREFIX$` files did not have newline at end, or at least according to this: https://community.bistudio.com/wiki/PBOPREFIX

Reply:
That's false. And we don't even use BI's Addon Builder anyways.

Reply:
BI Addon builder doesn't even use PBOPREFIX file.